magento/magento2
# Update colinmollenhour/credis library

### Description


### Fixed Issues

1. There is at least one bug in this library affecting Magento installations: https://github.com/magento/magento2/issues/9875, https://github.com/magento/magento2/issues/11110#issuecomment-332907514
2. There are new features in-demand by Magento developers: https://github.com/magento/magento2/issues/10962
3. Probably https://github.com/magento/magento2/issues/10524 will be fixed as well

### Manual testing scenarios

1. Configure Redis as session storage
2. Configure Redis as cache storage
3. Run some Selenium-based functional/acceptance tests to check that at least main Magento scenarios are not broken 

### Contribution checklist
